layers/+lang/plantuml/README.org

```diff
@@ -0,0 +1,28 @@
+#+TITLE: plantuml layer
+#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../css/readtheorg.css" />
+
+* Table of Contents                                        :TOC_4_org:noexport:
+ - [[Description][Description]]
+ - [[Install][Install]]
+ - [[Key bindings][Key bindings]]
+
+* Description
+This layer enables support for [[%5B%5Bfile:img/logo.png%5D%5D%0Ahttps://github.com/skuro/puml-mode][puml-mode]], which provides
```

Comment:
I'll fix it

--Rob
Sent from my cell, please excuse any typos.
On Jan 26, 2016 7:53 AM, "Fabien Dubosson" <notifications@github.com> wrote:

> In layers/+lang/plantuml/README.org
> <https://github.com/syl20bnr/spacemacs/pull/4795#discussion_r50830645>:
>
> > @@ -0,0 +1,28 @@
> > +#+TITLE: plantuml layer
> > +#+HTML_HEAD_EXTRA: <link rel="stylesheet" type="text/css" href="../../css/readtheorg.css" />
> > +
> > +* Table of Contents                                        :TOC_4_org:noexport:
> > + - [[Description][Description]]
> > + - [[Install][Install]]
> > + - [[Key bindings][Key bindings]]
> > +
> > +* Description
> > +This layer enables support for [[%5B%5Bfile:img/logo.png%5D%5D%0Ahttps://github.com/skuro/puml-mode][puml-mode] <http://github.com/skuro/puml-mode%5D%5Bpuml-mode%5D>], which provides
>
> %5B %5D?? Is this the github's rendering fault or do you really have this
> in your file instead of brackets?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/4795/files#r50830645>.
>